backend/src/test/java/de/neuefische/ffmjava221/teamprojekt/backend/employee/EmployeeServiceTest.java

```diff
@@ -39,4 +38,16 @@ void addEmployeeTEST() {
         //then
         assertEquals(employeeWithId, actual);
     }
+
+    @Test
+    @DirtiesContext
```

Comment:
DirtiesContext is used for integration tests, can be deleted

Reply:
ok removed